node/config/types.go

```diff
@@ -587,11 +583,6 @@ type EventsConfig struct {
 	// The API is enabled when Fevm.EnableEthRPC or EnableActorEventsAPI is true, but can be disabled selectively with this flag.
 	DisableRealTimeFilterAPI bool
 
-	// DisableHistoricFilterAPI will disable the HistoricFilterAPI that can create and query filters for actor events
-	// that occurred in the past. HistoricFilterAPI maintains a queryable index of events.
-	// The API is enabled when Fevm.EnableEthRPC or EnableActorEventsAPI is true, but can be disabled selectively with this flag.
-	DisableHistoricFilterAPI bool
```

Comment:
There's another one on line 563 that has a deprecation notice on it with a `moved:"Events.DisableHistoricFilterAPI"` tag - this tag will be interpreted by the config loader and try to look for the new property and not find it and probably fail with a weird error (if the user sets `Fevm.Events.DisableHistoricFilterAPI`).

Reply:
@rvagg Do we need to fix anything here ?

Reply:
Yeah, all those config options with `moved` in their tag which point to config fields that no longer exist are going to cause errors if someone tries to set them. So we need to remove them.

We've given enough notice so I think you should just remove the `DeprecatedEvents` struct entirely from here. It only existed as a transitional thing.

Reply:
This is done.